CMakeLists.txt

```diff
@@ -39,6 +39,11 @@ check_cxx_compiler_flag(-fopenmp-simd CXX_HAVE_OMP_SIMD)
 # Build options
 option(BUILD_DALI_NODEPS "Disable components that require extra external libraries to be present in the system. Effectively, it builds only the DALI core and kernel libraries")
 option(LINK_DRIVER "Links directly with libcuda.so instead of dlopen it at runtime" OFF)
+option(WITH_DYNAMIC_CUDA_LIBS "Links CUDA libraires dynamically (NPP, nvJPEG, cuFFT)" OFF)
```

Comment:
How does `WITH_DYNAMIC_CUDA_LIBS` option relate to `LINK_CUDA_DYNAMICALLY`? Can these two options be opposite (i.e. `WITH_DYNAMIC_CUDA_LIBS xor LINK_CUDA_DYNAMICALLY == 1`)?

Reply:
LINK_CUDA_DYNAMICALLY is about libcuda.so (driver) while WITH_DYNAMIC_CUDA_LIBS is about cuda toolkit. I will rename WITH_DYNAMIC_CUDA_LIBS to WITH_DYNAMIC_CUDA_TOOLKIT.